### DISCUSSION TITLE
这个项目需要如何编译？

### DISCUSSION THREAD

**Discussion by how2051** [xland/ScreenCapture#32] [Q&A]
习惯了Linux系统下Make/CMake编译，不是很清楚Windows下要怎么编译这个项目，我看README里没有提到，想知道需要怎么操作？

很希望后续能够参与到这个项目来

**Comment by xland** [comment]
1.先看看sln工程是怎么引入blend2d的
2.再用CMAKE在指定的目录下编译blend2d
